### PR TITLE
Feature flag HcbCode popovers

### PR DESCRIPTION
**Pull request by YodaLightsabr**

### CODE REVIEW

File: app/views/users/edit_featurepreviews.html.erb [hunk 1]
```diff
@@ -8,6 +8,23 @@
   <%= render "settings_nav", active: "previews" %>
   <section>
     <h2 id="feature-previews">Feature previews</h2>
+    
+    <% admin_tools("mb3") do %>
```
Comment (garyhtou) [inline app/views/users/edit_featurepreviews.html.erb]: No need to wrap in an admin tool, although it's okay if we do

**Review by garyhtou — CHANGES_REQUESTED** [review]
Let's also hide (don't render) the code for the popover modal itself

File: app/views/users/edit_featurepreviews.html.erb [hunk 2]
```diff
@@ -8,6 +8,21 @@
   <%= render "settings_nav", active: "previews" %>
   <section>
     <h2 id="feature-previews">Feature previews</h2>
+
+    <div class="card pb0" id="receipt-bin">
+      <div class="card__banner card__banner--top flex justify-between items-center feature-weekly-receipt-email feature-generic">
+        <h3 class="h1 p2 my0 color-black">HcbCode Popovers</h3>
```
Comment (garyhtou) [inline app/views/users/edit_featurepreviews.html.erb]:
```suggestion
        <h3 class="h1 p2 my0 color-black">Transaction Popovers</h3>
```

File: app/views/users/edit_featurepreviews.html.erb [hunk 3]
```diff
@@ -8,6 +8,21 @@
...
+        <% if Flipper.enabled?(:hcb_code_popovers_2023_06_16, current_user) %>
+          <%= link_to "Disable", disable_feature_users_path(feature: :hcb_code_popovers_2023_06_16), method: :post, class: "btn bg-accent" %>
+        <% else %>
+          <%= link_to "Enable", enable_feature_users_path(feature: :hcb_code_popovers_2023_06_16), method: :post, class: "btn bg-info" %>
+        <% end %>
+      </div>
+      <p>
+        This feature flag enables a popover allowing quick and easy access to transaction details from the ledger.
```
Comment (garyhtou) [inline app/views/users/edit_featurepreviews.html.erb]:
```suggestion
        This feature flag enables a popover allowing quick and easy access to transaction details from the organization's ledger.
```

**Review by garyhtou — CHANGES_REQUESTED** [review]

**Review by garyhtou — APPROVED** [review]

File: app/views/users/edit_featurepreviews.html.erb (hunk 2 above)
Comment (YodaLightsabr) [reply app/views/users/edit_featurepreviews.html.erb]: Oh right, I missed that. Thanks!

**Pull request merged**